manifests/0000_20_kube-apiserver-operator_04_clusterrolebinding.yaml

```diff
@@ -9,3 +9,6 @@ subjects:
 - kind: ServiceAccount
   namespace: openshift-kube-apiserver-operator
   name: kube-apiserver-operator
+- kind: ServiceAccount
+  name: localhost-recovery-client
```

Comment:
@deads2k the client certs were issued for group "system:masters", so binding it for cluster-admin seems equivalent, but I'd prefer if you sign it :)

Reply:
> @deads2k the client certs were issued for group "system:masters", so binding it for cluster-admin seems equivalent, but I'd prefer if you sign it :)

It's not the same.  If you lose an etcd connection, `system:masters` will continue to work, this will not.

Reply:
This binding logically belongs to the operator, not the CVO. Please move it so we can successfully change it in the future.

Reply:
you are right. I've moved it to the operator's bindata